### PR TITLE
Integrate Resource Page

### PR DESCRIPTION
**Pull request by lucindan**
-changed links on the modal for /resources to go to /resources/<id>
-added API method to search by resource._id

### CODE REVIEW

File: api/routes/resource.routes.js [hunk 1]
```diff
@@ -37,4 +37,9 @@ module.exports = (app) => {
   app.put('/api/resources/:_id', async (req, res) => {
     return await resourceUtil.update(req.params, req.body);
   });
+
```
Comment (sshh12) [inline api/routes/resource.routes.js:40]: Thanks for adding these!

File: src/views/ViewResource.js [hunk 2]
```diff
@@ -17,22 +17,22 @@ import {
...
 const { Panel } = Collapse;
-const props = {
-  name: 'Living Dictionary',
-  desc:
-    'An interactive dictionary of technical computer science and social science terms in plain language',
-  type: ['Education Tool'],
-  path: ['Explorer Path'],
-  avatarIcon:
-    'https://cdn.substack.com/image/fetch/w_170,c_limit,f_auto,q_auto:best,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F3fcac70c-a37c-4152-a109-f702c6375c31_256x256.png',
+const emptyRes = {
```
Comment (sshh12) [inline src/views/ViewResource.js:24]: can you remove everything that we no longer need here

File: src/views/ViewResource.js [hunk 3]
```diff
@@ -94,12 +100,24 @@ function FileTable(props) {
 }
 
 export default function ViewResource() {
+  let [resource, setResource] = useState(emptyRes);
```
Comment (sshh12) [inline src/views/ViewResource.js:103]: can we make this default to null (rather than empty res) and then just render a spinner if it hasn't loaded yet

**Review by sshh12 — CHANGES_REQUESTED** [review]
Some rly small tweaks

**Comment by lucindan** [comment]
spinner while loading

**Review by sshh12 — APPROVED** [review]

**Pull request merged**